scooter_manipulation/scripts/test_go_to_joint_config.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.action import ActionClient
+from rclpy.node import Node
+from action_msgs.msg import GoalStatus
+from scooter_interfaces.action import GoToJointConfig
+
+
+class JointConfigs:
+    fold_config = [0.0263192318379879, 1.2939172983169556, -1.7775753180133265, -1.2353704611407679, -1.5866463820086878, 0.1328919529914856]
```

Comment:
This should probably use the joint_configs.py JointConfigs() Class instances rather than the copy and paste. That way in the future if we update the class this test will still work properly

Reply:
This is a good point, I never really even intended to commit this file but I can update this (and document it) since it serves as a good example